libraries/adaptive-expressions/tests/expressionParser.test.js

```diff
@@ -621,6 +621,8 @@ const dataSource = [
     ['coalesce(nullObj, false, \'hello\')', false],
     ['jPath(jsonStr, pathStr )', ['Jazz', 'Accord']],
     ['jPath(jsonStr, \'.automobiles[0].maker\' )', ['Nissan']],
+    ['string(merge(json1, json2))', '{"FirstName":"John","LastName":"Smith","Enabled":true,"Roles":["User","Admin"]}'],
+    ['string(merge(json1, json2, json3))', '{"FirstName":"John","LastName":"Smith","Enabled":true,"Roles":["User","Admin"],"age":36}'],
```

Comment:
>"Roles":["User","Admin"] [](start = 98, length = 24)

Is this that last definition wins?  Or is this actually merging elements in an array?

Reply:
I changed the test case to make the more clear on merging arrays. The property in the latter input will replace the same property in the previous JSON object.